include/inviwo/core/util/stringconversion.h

```diff
@@ -155,6 +155,12 @@ constexpr void forEachStringPart(std::string_view str, std::string_view sep, Fun
     }
 }
 
+inline std::pair<std::string_view, std::string_view> divideBy(std::string_view str,
```

Comment:
splitBy would be more consistent with the naming of splitString

Reply:
True. But I considered that to be to similar, since splitString also takes a separator... I should add comment though...

Reply:
renamed to splitByFirst and splitByLast